src/frontend/src/components/Header/MobileHeader.jsx

```diff
@@ -78,6 +81,26 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
+function HideOnScroll(props) {
```

Comment:
This code is repeated.  Let's move it to a separate file so you can re-use the component in multiple files.

Reply:
Okay I just moved HideOnScroll to its own component. :)